tpm2/constants.go

```diff
@@ -236,6 +236,31 @@ const (
 	PermanentLast      TPMProp = 0x4000010F
 )
 
+const (
+	// Fixed TPM Properties (PT_VAR)
+	TPMAPermanent     TPMProp = 0x200 + iota
+	TPMAStartupClear
+	HRNVIndex
+	HRLoaded
+	HRLoadedAvail
+	HRActive
+	HRActiveAvail
+	HRTransientAvail
+	CurrentPersistent
+	AvailPersistent
+	NVCounters
+	NVCountersAvail
+	AlgorithmSet
+	LoadedCurves
+	LockoutCounter
+	MaxAuthFail
+	LockoutInterval
+	LockoutRecovery
+	NVWriteRecovery
+	AuditCounter0
+	AuditCounter1
+)
```

Comment:
I agree that we want to add these, but we should format this section in a different way (both for Godoc and for people reading the code).

First, change the statement for `TPMProp` to make it clear what this property actually is:
```go
// TPMProp represents a Property Tag (TPM_PT) used with calls to GetCapability(CapabilityTPMProperties).
type TPMProp uint32
``` 
Next, we have a section describing the Fixed TPM Properties:
```go
// TPM Capability Properties, see TPM 2.0 Spec, Rev 1.38, Table 23.
// Fixed TPM Properties (PT_FIXED)
const (
	FamilyIndicator TPMProp = 0x100 + iota
	...
	CapabilityMaxBufferSize
)
```
Then, we have a section describing the Variable TPM Properties:
```go
// Variable TPM Properties (PT_VAR)
const (
	TPMAPermanent     TPMProp = 0x200 + iota
	...
	AuditCounter1
)
```
Finally, a section for things we gave type `TPMProp` incorrectly:
```go
// Allowed ranges of different kinds of Handles (TPM_HANDLE)
// These constants have type TPMProp for backwards compatibility.
const (
	PCRFirst           TPMProp = 0x00000000
	...
	PermanentLast      TPMProp = 0x4000010F
)

```